<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="epigrass.qrc">
    <normaloff>:/egicon.png</normaloff>:/egicon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="statusTip">
        <string>Map visualization window</string>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="cacheMode">
        <set>QGraphicsView::CacheBackground</set>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="dataPanel">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Nodes">
        <property name="toolTip">
         <string>Node set</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <attribute name="title">
         <string>Nodes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="nodeTable">
           <property name="statusTip">
            <string>Node list.</string>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Edges">
        <attribute name="title">
         <string>Edges</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="edgeTable">
           <property name="statusTip">
            <string comment="Right-click for context menu">List of edges</string>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="actionDot"/>
     <addaction name="actionGML"/>
     <addaction name="actionPajek"/>
     <addaction name="actionYAML"/>
    </widget>
    <addaction name="actionOpen_edges_file"/>
    <addaction name="actionOpen_Shapefile"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="menu_Export"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Graph"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>&amp;Analyze</string>
    </property>
    <widget class="QMenu" name="menu_Debug">
     <property name="title">
      <string>&amp;Debug</string>
     </property>
     <addaction name="actionCheck_Graph"/>
    </widget>
    <addaction name="menu_Debug"/>
    <addaction name="separator"/>
    <addaction name="actionStats"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_edges_file"/>
   <addaction name="actionOpen_Shapefile"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Graph"/>
   <addaction name="separator"/>
   <addaction name="actionE_xit"/>
  </widget>
  <action name="actionOpen_edges_file">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/openedges.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open &amp;edges file</string>
   </property>
  </action>
  <action name="actionOpen_Shapefile">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/openshape.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open &amp;Shapefile</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/document-save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/document-save-as.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save &amp;as</string>
   </property>
  </action>
  <action name="action_Graph">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/graph.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Graph</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="icon">
    <iconset>
     <normalon>:/neteditor/application-exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCheck_Graph">
   <property name="text">
    <string>Check Graph</string>
   </property>
  </action>
  <action name="actionDot">
   <property name="text">
    <string>Dot</string>
   </property>
  </action>
  <action name="actionGML">
   <property name="text">
    <string>GML</string>
   </property>
  </action>
  <action name="actionPajek">
   <property name="text">
    <string>Pajek</string>
   </property>
  </action>
  <action name="actionYAML">
   <property name="text">
    <string>YAML</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="epigrass.qrc"/>
 </resources>
 <connections/>
</ui>
